insertion_sort/C/insertionsort.c

```diff
@@ -0,0 +1,30 @@
+// Program implementing insertion sort using array in C
+#include<stdio.h>
+// Function to sort an array using insertion sort
+void insertionsort(int ar_size, int *ar) {
+    int temp, j;
+    for (int i = 1; i < ar_size; i++) {
+        j = i;
+        temp = ar[j];
+    /* Shift elements by one position ahead of its current position
+       that are greater than temp(the element being sorted). */
+        while (j > 0 && temp < ar[j-1]) {
+            ar[j] = ar[j-1];
+            j--;
+        }
+    ar[j] = temp;
+}
+
+
+int main() {
+    int a[9] = {5, 8, 12, 15, 9, 1, 3, 7, 24};
+    int n = sizeof(a) / sizeof(a[0]);  // Counting number of array elements.
+
+    insertionsort (n, a);  // Calling function for insertion sorting.
+
+    for (int i = 0; i < n; i++)  // Printing the array
+        printf ("%d ", a[i]);
```

Comment:
Extra space before ( in function call  [whitespace/parens] [4]

*Origin: CPPLintBear, Section: `all.cpp`.*